### PR TITLE
Unnecessary downloading eliminated

### PR DESCRIPTION
**Pull request by batuhanfaik**
Doesn't redownload the files anymore
Doesn't create duplicated folders
Doesn't overwrite files with the same name (keeps previous versions of files)
Variable overwrite is defined as global so any method can check its value

To-do: 

- Date comparison between files

- Ask the user whether to update when new versions of files available on Ninova

- Silent mode (update everything to newest, preserve current files, don't require input from the user)

### CODE REVIEW

File: ninova_downloader.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import os
-import datetime
-import shutil
+
+import bs4
+import requests
 
 '''Base URL'''
 url = 'https://ninova.itu.edu.tr'
+overwrite = 'n'
```
Comment (erfur) [inline ninova_downloader.py]: Wouldn't it be better to use boolean values for this?
```suggestion
overwrite = False
```

File: ninova_downloader.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
-import requests
-import bs4
+import datetime
 import getpass
 import os
-import datetime
-import shutil
+
+import bs4
+import requests
```
Comment (erfur) [inline ninova_downloader.py]: Let's not change irrelevant parts of the code and cause unnecessary additions/deletions.

File: ninova_downloader.py (hunk 1 above)
Comment (batuhanfaik) [reply ninova_downloader.py]: Sure why not

File: ninova_downloader.py (hunk 2 above)
Comment (batuhanfaik) [reply ninova_downloader.py]: Don't mind that. It's just the IDE doing import optimizations

**Pull request merged**